### PR TITLE
Project assessment

### PR DESCRIPTION
**Pull request by jasonburt**
search add record was overriding, was a broken open. Fixed that and some quality of life issues.

### CODE REVIEW

File: src/cli/helpers.py [hunk 1]
```diff
@@ -217,15 +218,27 @@ def record_struct(name, search, search_type, record_links, save, category, subca
     language = "general"
     name = re.sub("[^A-Za-z0-9]+", "", name)
     name = name + "_file_check"
-    save_path = Path("assessments") / save / "evidence.json"
+    if environment == 'production':
+        save_path = Path("assessments/"+save+"/evidence.json")
```
Comment (alejones) [inline src/cli/helpers.py:222]: Just a nitpick, to concatenate things with a Path object you shouldn't use + use / 
```python
save_path = Path("assessments") / save / "evidence.json"
```

File: src/cli/helpers.py [hunk 2]
```diff
@@ -217,15 +218,27 @@ def record_struct(name, search, search_type, record_links, save, category, subca
...
+    
     # save_path = "/rubric/" + language + "/" + name + ".json"
 
     # Load records
     try:
-        with open(save_path, "r") as records_file:
-            records = json.loads(records_file)
+        with open(save_path, "r")as records_file:
+            records_data = records_file.read()
+        records = json.loads(records_data)
```
Comment (alejones) [inline src/cli/helpers.py:233]: I believe `json.loads` is to load a json string, use `json.load` to load a json from a file.

File: src/cli/helpers.py [hunk 3]
```diff
@@ -217,15 +218,27 @@ def record_struct(name, search, search_type, record_links, save, category, subca
...
+        print(f"Existing Records: {len(records)}")
     except:
```
Comment (alejones) [inline src/cli/helpers.py:235]: I would change this to 

```python
try:
    ....
except FileNotFoundError as fnf_errorr:
    print(f"FileNotFoundError:'{save_path}'.")
```

File: src/cli/helpers.py (hunk 1 above)
Comment (jasonburt) [reply src/cli/helpers.py:222]: Thanks will make the change on the next one / future.

File: src/cli/helpers.py (hunk 2 above)
Comment (jasonburt) [reply src/cli/helpers.py:233]: Was getting an issue with PATH being defined and being passed into load. It errors out. This works for now.

**Comment by jasonburt** [comment]
@alejones : Thanks for reviewing! Lots of great points. If its minor things can you review and approve? For flow would like to be able to do those minor updates in the future.

**Comment by jasonburt** [comment]
@alejones : I saw it was opened and closed. Do you see the review link in the Review required section? Just asking as I had changed some of the repo settings for review approval and wanted to make sure its visibile to you.

**Review by reed271 — APPROVED** [review]

**Comment by alejones** [comment]
I had some company here all weekend, just getting back around to this. I fat fingered the close button when I was on my phone. I can start approving after making any comments for branches.

**Pull request merged**